app/models/container_project.rb

```diff
@@ -47,6 +50,10 @@ def containers_count
     container_definitions.size
   end
 
+  def images_count
```

Comment:
could you instead use virtual_total :images_count, :container_groups

Also for naming, think we tend to use a name like `:total_groups`